contracts/Exchanger.sol

```diff
@@ -134,22 +134,34 @@ contract Exchanger is IExchanger {
         // Estimate "neededOUSD" using get_dy()
         uint256 _needed3CRV = _poolLvUSD3CRV.get_dy(_indexLvUSD, _index3CRV, minRequiredLvUSD);
         uint256 _neededOUSD = _poolOUSD3CRV.get_dy(_index3CRV, _indexOUSD, _needed3CRV);
-        _needed3CRV = (_neededOUSD * 103) / 100;
+        uint256 _neededOUSDWithSlippage = (_neededOUSD * 101) / 100;
```

Comment:
Why 1% and not 3% or 5%? I'm concerned that if there is high volatility, that low slippage might cause lots of reverts and prevent exchanges. I don't know what the best slippage should be though.